src/js/page/ui/settings.js

```diff
@@ -67,14 +67,32 @@ class Settings extends (require('events').EventEmitter) {
 
   }
 
+  setSettings(settings) {
+    this._globalInputs.forEach(function(inputEl) {
+      if (inputEl.type == 'checkbox') {
+        inputEl.checked = settings[inputEl.name];
+      }
+      else if (inputEl.type == 'range') {
+        inputEl.value = settings[inputEl.name];
+        var rangeChangeEvent = utils.isIe ? 'change' : 'input';
+        var event = new Event(rangeChangeEvent);
+        inputEl.dispatchEvent(event);
```

Comment:
This is only for the range - just updating the input didn't update the material-slider elements so ui was out of sync